unit-tests/log/test-all.py

```diff
@@ -0,0 +1,25 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+from rspy import log, test
+import pyrealsense2 as rs
+import common
+
+
+#############################################################################################
+#
+test.start( f"log all" )
```

Comment:
`f` is used to format strings no? with parameters?

Reply:
They're called f-strings in Python, for formatting. Here there isn't any, but it does not hurt...